packages/office-ui-fabric-react/src/common/_a11y.scss

```diff
@@ -0,0 +1,11 @@
+// To hide content while still making it readable by screen readers (Accessibility)
+@mixin ms-u-screenReaderOnly {
```

Comment:
This mixin is already included in Fabric Core: https://github.com/OfficeDev/office-ui-fabric-core/blob/master/src/sass/mixins/_Utility.Mixins.scss#L52